### DISCUSSION TITLE
Click through

### DISCUSSION THREAD

**Discussion by paulosimao** [hajimehoshi/ebiten#1909] [Q&A]
Hello, I would like to use ebiten in a way it will draw on top of the screen, but still allowing user to interact with whatever GUI components are below the transparent window - click through is what they call it. Is it possible to be achieved w Ebiten, at least in Windows?

Thanks!

**Comment by hajimehoshi** [comment]
Hi,

It is impossible to do that, and unfortunately this would not be easy to implement such feature. Is there an actual application to do that by the way?

Thanks!

**Reply by Zyko0** [reply]
I also don't think it's something obvious, but you could probably record a click and x,y cursor position on your ebiten window and forward it to the window behind with some Windows API calls.
Assuming your ebiten app would be an overlay for always the same specific window (which you know the handler id).

But yes no ebiten feature can allow that for now, most of the work would be outside ebiten scope

**Reply by paulosimao** [reply]
Thanks you guys! I will try to figure it out. If I find it, will let you know. KR!
